Exiled.API/Features/Player.cs

```diff
@@ -1933,15 +1990,37 @@ public void Kill(string deathReason, string cassieAnnouncement = "")
             ReferenceHub.playerStats.KillPlayer(new CustomReasonDamageHandler(deathReason, float.MaxValue, cassieAnnouncement));
```

Comment:
I know it's not from your pr but this should be -1

Reply:
It's better in case of NW changing value and using double instead of float for example

Reply:
yeah yeah I was just making sure that's what you wanted me to change